<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Login">
      <property name="styleSheet">
       <string notr="true">QFrame{
border:0;
background:black;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="TOP">
         <property name="styleSheet">
          <string notr="true">border:0;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="MAIN">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>500</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:0;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="MAIN_2">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>400</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>400</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color: #333;
border-radius: 20%;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLineEdit" name="entryPwd">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>150</y>
               <width>201</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>MS UI Gothic</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::TabFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
background: #333;
border: none;
border-bottom: 1px solid white;
color: white;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
             <property name="cursorPosition">
              <number>0</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>Servidor</string>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::LogicalMoveStyle</enum>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLabel" name="titulo">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>10</y>
               <width>240</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI Semibold</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
border: 0;
</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="text">
              <string>CNS - Consulta</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>50</y>
               <width>91</width>
               <height>91</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>img/icons8-lupa-48.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLineEdit" name="entryPwd_2">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>190</y>
               <width>201</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>MS UI Gothic</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::TabFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
background: #333;
border: none;
border-bottom: 1px solid white;
color: white;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
             <property name="cursorPosition">
              <number>0</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>Usu√°rio</string>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::LogicalMoveStyle</enum>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLineEdit" name="entryPwd_3">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>230</y>
               <width>201</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>MS UI Gothic</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::TabFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
background: #333;
border: none;
border-bottom: 1px solid white;
color: white;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::PasswordEchoOnEdit</enum>
             </property>
             <property name="cursorPosition">
              <number>0</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>Senha</string>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::LogicalMoveStyle</enum>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QRadioButton" name="radioButton">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>290</y>
               <width>91</width>
               <height>17</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
border: 0;</string>
             </property>
             <property name="text">
              <string>Salvar Usuario</string>
             </property>
             <property name="shortcut">
              <string>Ctrl+S</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btnLogin">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>340</y>
               <width>101</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>MS UI Gothic</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background-color: black;
	border-radius: 20px;
	color: white;
}</string>
             </property>
             <property name="text">
              <string>Login</string>
             </property>
             <property name="shortcut">
              <string>Enter</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="BOT">
         <property name="styleSheet">
          <string notr="true">border:0;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
